### DISCUSSION TITLE
How to enable GPU computing?

### DISCUSSION THREAD

**Discussion by njzjz** [dftbplus/dftbplus#935] [Q&A] [answered]
Dear developers,
In your [recent paper](https://aip.scitation.org/doi/full/10.1063/1.5143190), it's mentioned that DFTB+ supports GPU computing. However, I didn't find how to enable it. Can you give me some tips?
Jinzhe

**Comment by aradi** [comment, marked as answer]
You have to compile the code with GPU-support (`-DWITH_GPU=1`) Then you select the MAGMA solver (`Solver = Magma {}`)

**Reply by njzjz** [reply]
I got an error when compiling with MAGMA:
```
$SRC_DIR/src/dftbp/extlibs/magma.F90:14:6:

   14 |   use magma, only : magmaf_ssygvd_m, magmaf_dsygvd_m, magmaf_chegvd_m, magmaf_zhegvd_m                                                            |      1
Fatal Error: Cannot open module file 'magma.mod' for reading at (1): No such file or directory                                                      compilation terminated.
```

Where is `magma.mod`? I cannot find it anywhere

**Reply by aradi** [reply]
Probably, your MAGMA library was compiled without Fortran support. Make sure, the Fortran support is turned on, when compiling MAGMA (e.g. by passing `-DUSE_FORTRAN=on` to cmake when building the MAGMA library itself).
